ircbot/plugins/currency.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from urllib.request import urlopen
+
+from ircbot import bot
+
+
+def get_currency(currency_a, currency_b):
```

Comment:
Could you document this a bit more, or change it to a more descriptive name? Just from reading the code, I would guess "currency ratio" would be the exchange rate?